packages/ember-headless-form/src/-private/components/control/radio-group.gts

```diff
@@ -0,0 +1,87 @@
+import { hash } from '@ember/helper';
+
+import { uniqueId } from '../../utils';
+import HeadlessFormControlRadioGroupLabelComponent from './radio-group/label';
+import HeadlessFormControlRadioComponent from './radio-group/radio';
+
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import type { WithBoundArgs } from '@glint/template';
+
+export interface HeadlessFormControlRadioGroupComponentSignature {
+  Element: HTMLDivElement;
+  Args: {
+    // the following are private arguments curried by the component helper, so users will never have to use those
```

Comment:
ah, I like this comment and the `@internal` syntax.  I ran into this with a component recently and was wondering if there was good messaging like this